Riot/Modules/HealthServices/Data/Role.swift

```diff
@@ -22,22 +22,60 @@ class Role: Equatable {
     }
     
     let Name: String
+    let Title: String
     let Identifier: String
-    let Description: String
+    let OfficialName: String
+    
+    //Category, Org Unit, Location and Designation should in reality not be strings but instead links to other objects
     let Designation: String
+    
+    let Location: String
+    //is "Category" actually "Specialty"?
+    let Category: String
+    let OrgUnit: String
+    
     init(withName n: String, andId id: String, andDescription desc: String) {
         Name = n
+        Title = n
         Identifier = id
-        Description = desc
+        OfficialName = desc
         Designation = n
+        //a location string should come about by looking up the location, and then generating the abbreviated string.
+        Location = "CH {Canberra Hospital}"
+        Category = "Emergency"
+        OrgUnit = "ED {Emergency Department}"
     }
     init(withName n: String, andId id: String, andDescription desc: String, andDesignation des: String) {
         Name = n
+        Title = n
```

Comment:
To centralise code can this init call the other init method e.g.
init(n, id, desc, des, "CH {Canberra Hospital}")

Reply:
Updated these initialisers to be convenience init methods, used them to call the original init method.